runtime/plugin/man.vim

```diff
@@ -5,9 +5,9 @@ if exists('g:loaded_man')
 endif
 let g:loaded_man = 1
 
-command! -bang -bar -range=0 -complete=customlist,man#complete -nargs=* Man
-      \ if <bang>0 | set ft=man |
-      \ else | call man#open_page(v:count, v:count1, <q-mods>, <f-args>) | endif
+command! -bang -bar -range=-1 -complete=customlist,man#complete -nargs=* Man
```

Comment:
I liked the way we were doing things before as it was very explicit in when `v:count` was actually set. What if at some point someone does make a `-1` manpage section?

Reply:
The count wasn't working with, e.g., `:3Man printf` would produce the man page for `printf(1)` instead of `printf(3)` (for some mysterious reason it did work with `K`). Also I think limiting the count to non-negative numbers simplifies the code, and consider that there exists (or might exist) man sections which are not numeric (see [wikipedia](https://www.wikiwand.com/en/Man_page#/Manual_sections)) and so we cannot support them with counts or ranges.